contrib/babelfishpg_tsql/src/rolecmds.c

```diff
@@ -684,6 +685,10 @@ user_id(PG_FUNCTION_ARGS)
 	if (!user_name)
 		PG_RETURN_NULL();
 
+	if (pltsql_case_insensitive_identifiers)
+        // Lowercase the entry, if needed
+        for (char *p = user_name ; *p; ++p) *p = tolower(*p);
```

Comment:
Please fix the indentation.

Why can't we use downcase_identifier?

Reply:
I had an offline conversation with Sangil about this with the previous fix https://code.amazon.com/reviews/CR-69414431/revisions/2#/comments - downcase_identifier caused segfaults in some scenarios when being used here.